results/cross_encoder__ms-marco-TinyBERT-L-2-v2/News21InstructionRetrieval.json

```diff
@@ -0,0 +1,87 @@
+{
+  "dataset_revision": "fae860d870141ee0f34fbab67314c65f41921943",
+  "first_stage_model": "SentenceTransformer (unknown)",
+  "mteb_dataset_name": "News21InstructionRetrieval",
+  "mteb_version": "1.6.34",
+  "test": {
+    "evaluation_time": 87.08,
+    "individual": {
+      "base": {},
+      "changed": {
```

Comment:
This file is a little misleading, this is actually the InstructionRetrieval results, so that's where the changed and original come from.  It's here just to show that the PR works for both `Retrieval` and `InstructionRetrieval`

Reply:
Yea, this doesn't really fit into this PR, it was a general comment (but let us keep it out of this PR for now)